lib/webhook.js

```diff
@@ -1,15 +1,15 @@
-module.exports = function(parser, fixer, commiter) {
+module.exports = function(parser, fixer, commiter, githubapi) {
     return function *webhook(next) {
         var request = this.request;
 
         var parsedContent = parser(request);
-        var fixedContent = fixer(parsedContent);
+        var fixedContent = yield fixer(parsedContent, githubapi);
```

Comment:
Please use verbs but not the infinitive. Your code would be much more readable like that:

```js
var parsedContend = parse(request);
var fixedContent = yield fix (parsedContent, githubapi);
```